app/preview.py

```diff
@@ -63,7 +64,18 @@ def get_page_count(pdf_data):
 @auth.login_required
 def page_count():
     json = get_and_validate_json_from_request(request, preview_schema)
-    return jsonify({"count": get_page_count(get_pdf(get_html(json)).read())})
+    if json["template"]["letter_attachment"]:
+        attachment_page_count = json["template"]["letter_attachment"]["page_count"]
+    else:
+        attachment_page_count = 0
+    page_count = get_page_count(get_pdf(get_html(json)).read()) + attachment_page_count
+    return jsonify({"count": page_count})
```

Comment:
When a user uploads an attachment they will go to the page where they see the whole letter.

I would like it so they are jumped down the page so that the first thing they see is the first page of the attachment. This will give them confidence that the upload has worked.

I’m happy to make this change in the admin app – it’s not a big one. But it relies on the admin app knowing on which page the attachment starts.

Reply:
Ahh ok makes sense, thanks for explaining 👍🏼

Reply:
I will make the change you suggested, but will think of the naming some more 👍🏼

Reply:
I added `attachment_page_count` in the end, as `original_page_count` sounds a bit confusing, and might get even more so when Welsh templates come in.

Ideally `count` key should change to `total_page_count`. Might do it as a separate thing, since it's a bit of a dance to make this change.

Reply:
Nice, that will work well. Agree that `total_page_count` would be clearer, but fine to make as a separate change.